PACKAGES.md

```diff
@@ -0,0 +1,25 @@
+# Packages
+
+## Updating source code
+
+1. Begin by committing modifications to the github repository that hosts the source for your intended nix package
+
+2. Make note of your commit reference
+
+## Updating default.nix
+
+1. Change the `rev` field in `fetchFromGitHub` to your desired commit reference
+
+2. Run the command `nix-prefetch-url --unpack <Github commit archive URL>.tar.gz`
```

Comment:
I wonder if we shouldn't give an example of how an archive URL is formed. I know it wasn't apparent to me prior to working with Nix and would expect others to probably not know as well.

Reply:
I was thinking about this and chose not to so as to not bloat the docs. But yeah it would probably be a good idea

I'll add that